packages/metals-vscode/src/extension.ts

```diff
@@ -195,22 +188,31 @@ function fetchAndLaunchMetals(
   /* eslint-enable @typescript-eslint/no-non-null-assertion */
 
   const coursierMirror = getCoursierMirrorPath(config);
+
+  const metalsDirPath = path.join(os.homedir(), ".metals");
```

Comment:
```suggestion
  const metalsDirPath = metalsDir(ConfigurationTarget.Global);
```
maybe? we already have it in utils